install_files/ansible-base/roles/app-test/tasks/main.yml

```diff
@@ -6,3 +6,5 @@
   tags: non-development,aa-complain
 
 - include: dev_setup_xvfb_for_functional_tests.yml
+
+- include: security_hardening_tests.yml
```

Comment:
For now we can just add ` tags: non-development` to the `security_hardening_tests.yml` include. Down the road we should refactor the ansible config to leverage hostgroups so we can bundle up excludes like this for travis and other environments.